### PR TITLE
tests: run a long test

### PR DESCRIPTION
**Pull request by JohnMcLear**
Designed to catch issues that only appear X duration.

We might want to introduce some "chaos" into the load test tool or do we think this will be sufficient?

### CODE REVIEW

File: .github/workflows/load-test.yml [hunk 1]
```diff
@@ -78,6 +78,34 @@ jobs:
     - name: Install all dependencies and symlink for ep_etherpad-lite
       run: src/bin/installDeps.sh
 
-    # configures some settings and runs npm run test
+    # configures some settings and runs npm run test for 25 seconds with 50 authors
```
Comment (rhansen) [inline .github/workflows/load-test.yml:81]: This comment is likely to drift out of sync with the code. I'd say leave out the "for 25 seconds with 50 authors". If you want it to be more readable then use more verbose flags like `--duration=25 --users=50`.

**Review by rhansen — APPROVED** [review]
Leave out chaos for now. I agree that it would be good to introduce chaos in the future, but that's really hard to do properly: A general rule about testing is it should be easy to reproduce a failure. That's difficult when there's randomness or extreme complexity.

File: .github/workflows/load-test.yml (hunk 1 above)
Comment (JohnMcLear) [reply .github/workflows/load-test.yml:81]: why would it drift out of sync?

Comment (rhansen) [reply .github/workflows/load-test.yml:81]: It's common for people to bump a number and forget to update the comment to match. Then that leaves people scratching their heads.

File: .github/workflows/load-test.yml [hunk 2]
```diff
@@ -80,4 +80,32 @@ jobs:
...
+        node-version: 12
+
+    - name: Install all dependencies and symlink for ep_etherpad-lite
+      run: src/bin/installDeps.sh
+
+    - name: Install etherpad-load-test
+      run: sudo npm install -g etherpad-load-test
+
+
+    # configures some settings and runs npm run test for 5000 seconds(50 mins) with 5 authors
```
Comment (JohnMcLear) [inline .github/workflows/load-test.yml:109]: What about this one @rhansen  ?

Comment (rhansen) [reply .github/workflows/load-test.yml:109]: Delete it.

**Pull request merged**